packages/cli/test/unit/util/dev/validate.test.ts

```diff
@@ -229,7 +229,23 @@ describe('validateConfig', () => {
       'Invalid vercel.json - `headers[1].headers` should NOT have more than 1024 items.'
     );
     expect(error!.link).toEqual(
-      'https://vercel.com/docs/configuration#project/headers'
+      'https://vercel.com/docs/concepts/projects/project-configuration#headers'
+    );
+  });
+
+  it('should error with invalid memory value', async () => {
+    const error = validateConfig({
+      functions: {
+        'api/test.js': {
+          memory: 1000,
```

Comment:
I thought 1000 is valid now

Reply:
Not until the the schema is updated 😜
But yeah makes sense to use another value so I don't have to update it right after the schema change lands^^